src/backend/functions/setup/ecr-handler.js

```diff
@@ -0,0 +1,6 @@
+const { REPOSITORY } = process.env;
+
+module.exports = ecr => ({
+  removeRepository: () =>
+    ecr.deleteRepository({ force: true, repositoryName: REPOSITORY }).promise()
```

Comment:
Similar to S3, to delete a stack we need ECR to be empty. With force: true the repo deletes all the images inside.

Reply:
Forget this, delete repository was actually making the delete cfn fail, so I now list the images, and then delete all of them, leaving the repository deletion to cfn 💃